recipes/aws-sdk-cpp/run_test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Stop on first error
+set -e
+
+if [ "$(uname)" == "Darwin" ]; then
+    # OSX
+    g++ test.cpp -I"${PREFIX}/include" -L"${PREFIX}/lib" -laws-cpp-sdk-core -o test-core
```

Comment:
Change this `clang++ test.cpp -I"${PREFIX}/include" -L"${PREFIX}/lib" -Wl,-rpath,${PREFIX}/lib -laws-cpp-sdk-core -o test-core`